docs/source/release_notes.rst

```diff
@@ -7,6 +7,9 @@ Release Notes
         * Set ``eval_metric`` to ``logloss`` for ``XGBoostClassifier`` :pr:`2741`
         * Added support for ``woodwork`` versions ``0.7.0`` and ``0.7.1`` :pr:`2743`
         * Changed ``explain_predictions`` functions to display original feature values :pr:`2759`
+        * Added ``X_train`` and ``y_train`` to ``graph_prediction_vs_actual_over_time`` and ``get_prediction_vs_actual_over_time_data`` :pr:`2762`
+        * Added ``forecast_horizon`` as a required parameter to time series pipelines and ``AutoMLSearch`` :pr:`2697`
```

Comment:
Accidentally put the release notes for 2697 into the previous release during a rebase lol